examples/quickstart-pennylane/pyproject.toml

```
@@ -0,0 +1,56 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "quickstart_pennylane"
+version = "1.0.0"
+description = "Quantum Federated Learning with PennyLane and Flower"
+license = "Apache-2.0"
+authors = [
+    { name = "SECQUOIA Research Group", website = "https://secquoia.github.io/" },
+    { name = "David Bernal Neira", email = "dbernaln@purdue.edu" },
+    { name = "Daniel Anoruo", email = "danoruo1@students.towson.edu" },
+    { name = "Alan Yi", email = "yi161@purdue.edu" }
+]
+dependencies = [
+    "flwr[simulation]>=1.21.0",
+    "flwr-datasets[vision]>=0.5.0",
+    "torch==2.8.0",
+    "torchvision==0.23.0",
+    "pennylane==0.41.1",
+    "autoray>=0.6,<0.7",
+]
+
+[tool.hatch.build.targets.wheel]
+packages = ["."]
+
+[tool.flwr.app]
+publisher = "flwrlabs"
+
+[tool.flwr.app.components]
+serverapp = "quickstart_pennylane.server_app:app"
+clientapp = "quickstart_pennylane.client_app:app"
+
+[tool.flwr.app.config]
+num-server-rounds = 10
+batch-size = 32
+local-epochs = 2
+learning-rate = 0.001
+fraction-fit = 1.0
```

```suggestion
fraction-train = 1.0
```
The new strategies replace the `fit` term with `train`. We can update this (I've also left some related code suggestions in the `server_app.py`.